examples/mongodbatlas_advanced_cluster/asymmetric-sharded-cluster/README.md

```diff
@@ -0,0 +1,62 @@
+# MongoDB Atlas Provider -- Global Cluster 
+This example creates a project and a Sharded Cluster with 4 independent shards with varying cluster tiers and auto-scaling enabled.
```

Comment:
since the example is tailored to having specific tiers in each shard, not sure if I would include in this example the use of auto-scaling. WDYT?

Reply:
I added it because we currently don't have any examples with auto-scaling and it probably gives a good insight into the ISS capabilities too

Reply:
Double checking on this I see that auto-scaling for independent sharded cluster was not included as part of phase 1 (https://jira.mongodb.org/browse/CLOUDP-200550?focusedCommentId=5892546&focusedId=5892546&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-5892546), meaning if autoscaling is defined it scales all shards at the same time. I would keep this example specific to ISS to avoid any confusion for users.

Reply:
got it, thanks for sharing this, I'll revert, I took this from an example in the ISS spec

Reply:
removed